tests/usage.bats

```diff
@@ -43,24 +43,24 @@ EOF
 
 @test "f2: command grouping changes 1: 'Local Development Commands' listed in order" {
     run --separate-stderr "$FLOX_CLI" --help;
-    assert_line -n 1 --regexp '^Local Development Commands';
-    assert_line -n 2 --regexp '^    init[ ]+[\w .,]+';
-    assert_line -n 3 --regexp '^    activate[ ]+[\w .,]+';
-    assert_line -n 4 --regexp '^    search[ ]+[\w .,]+';
-    assert_line -n 5 --regexp '^    show[ ]+[\w .,]+';
-    assert_line -n 6 --regexp '^    install[ ]+[\w .,]+';
-    assert_line -n 7 --regexp '^    uninstall[ ]+[\w .,]+';
-    assert_line -n 8 --regexp '^    edit[ ]+[\w .,]+';
-    assert_line -n 9 --regexp '^    list[ ]+[\w .,]+';
-    assert_line -n 10 --regexp '^    delete[ ]+[\w .,]+';
+    assert_line -n 4 --regexp '^Local Development Commands';
+    assert_line -n 5 --regexp '^    init[ ]+[\w .,]+';
+    assert_line -n 6 --regexp '^    activate[ ]+[\w .,]+';
+    assert_line -n 7 --regexp '^    search[ ]+[\w .,]+';
+    assert_line -n 8 --regexp '^    show[ ]+[\w .,]+';
+    assert_line -n 9 --regexp '^    install[ ]+[\w .,]+';
+    assert_line -n 10 --regexp '^    uninstall[ ]+[\w .,]+';
+    assert_line -n 11 --regexp '^    edit[ ]+[\w .,]+';
+    assert_line -n 12 --regexp '^    list[ ]+[\w .,]+';
+    assert_line -n 13 --regexp '^    delete[ ]+[\w .,]+';
```

Comment:
can you do this with a variable? and then add +1 it with each line?

Reply:
Yeah, good idea